File: BattleSet/step7.rb
# あなたは奇数が大好きな山田さんと数字の仕分けをすることになりました。
# 仕分けをする数字が与えられますので、奇数であるものを小さい順に出力してください。

n = gets.to_i
numbers = []
n.times do 
    numbers << gets.to_i
end

odds = []
numbers.each do |i|
    if i.odd?
        odds << i
    end
end

puts odds.sort

File: Python/ConditionalBranch/ProductMinimization/step7.py
# 長さ N の数列Aが与えられます。この数列に含まれる偶数の要素の個数と、奇数の要素の個数を答えてください。

n  = int(input())
numbers = map(int,input().split())

odd = 0
even = 0

for i in numbers:
    if (i+1) % 2 == 0:
        even += 1
    else:
        odd += 1

print(odd,even)
